### DISCUSSION TITLE
@types/[echarts]  levels property on the Treemap and Sunburst series is falsely not an array

### DISCUSSION THREAD

**Discussion by ThanosSoulis** [DefinitelyTyped/DefinitelyTyped#58276] [Issues with a @types package] [answered]
On the [Treemap](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/12667667636cd8cb88c8073691d9498b1631c7ad/types/echarts/options/series/treemap.d.ts) and [Sunburst](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/12667667636cd8cb88c8073691d9498b1631c7ad/types/echarts/options/series/sunburst.d.ts) type definitions the levels property is not an Array as per the official documentation :
- https://echarts.apache.org/en/option.html#series-treemap.levels
- https://echarts.apache.org/en/option.html#series-sunburst.levels

I could try and make a pull request but I am not sure if there is a template to follow.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment, marked as answer]
It looks like this was fixed in #58164:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ec46713211797e8f01bef919113d4ec91458e0d2/types/echarts/options/series/treemap.d.ts#L4159

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ec46713211797e8f01bef919113d4ec91458e0d2/types/echarts/options/series/sunburst.d.ts#L8102
